inc/updater/class-updater.php

```diff
@@ -237,7 +237,7 @@ public function update_site_transient( $transient ) {
 
 		$rel_path = plugin_basename( $this->filepath );
 		$rel_path = 'theme' === $this->type ? dirname( $rel_path ) : $rel_path;
-		$response = Packages\get_update_data( $this->did );
+		$response = Packages\get_package_data( $this->did );
 		if ( is_wp_error( $response ) ) {
 			return $transient;
 		}
```

Comment:
Looks like the issue is here. Need to swap the `slug` for `slug_didhash`.

```
$response = Packages\get_package_data( $this->did );
if ( is_wp_error( $response ) ) {
	return $transient;
}
$response['slug'] = $response['slug_didhash'];
```

Reply:
I think I found the issue in `alter_slugs()`. Need to exit if plugin already installed.